### ISSUE TITLE
[VT-98] <del> key not working while editing command

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#120]
_David Patrick on 2014-06-27T15:30:31Z says:_

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-06-27T15:30:31Z
Modified: 2015-03-02T06:22:59Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-03-01T00:38:57Z says:_ 

 I can reproduce. Indeed, it doesn't work for any of VIT's prompts.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2015-03-02T06:22:59Z says:_ 

 Fixed at 3c30a83e.
